stablehlo/tools/StablehloTranslateMain.cpp

```diff
@@ -131,15 +150,23 @@ llvm::Error interpreterFallback(Operation &op, stablehlo::Scope &scope,
 TranslateFromMLIRRegistration interpretRegistration(
     "interpret", "Interpreter for StableHLO",
     [](ModuleOp module, raw_ostream &os) {
+      auto opsCount = 0;
```

Comment:
`numFuncs`? To follow this convention in MLIR which has functions like `getNumArguments`. (Same for `argsSize`).